README.md

```diff
@@ -0,0 +1,50 @@
+NOnion
+-------------------------------
+[![Build Status](https://github.com/nblockchain/NOnion/actions/workflows/CI.yml/badge.svg?branch=master&event=push)](https://github.com/nblockchain/NOnion/actions/workflows/CI.yml)
+
+_Unofficial_ work in progress TOR client library implemented in F#
+
+- [How do I add this to my project?](#how-do-i-add-this-to-my-project)
+- [Contributors](#contributors)
+- [License](#license)
+
+# How do I add this to my project?
+
+Install via NuGet
+
+```
+	> Install-Package NOnion
```

Comment:
please note, this is not a command that works in any terminal, let's add more specific instructions here